portal-web/test/functional/com/liferay/portalweb/macros/layout/sitepage/Page.macro

```diff
@@ -230,13 +229,12 @@ definition {
 
 		SelectFrame(locator1 = "IFrame#MODAL_BODY");
 
-		var key_cardText = "${themeName}";
-
 		AssertTextPresent(
-			locator1 = "Card#SPECIFIC_LARGE_TEXT",
-			value1 = "${key_cardText}");
+			key_card = "${themeName}",
+			locator1 = "LexiconCard#CARD",
+			value1 = "${themeName}");
```

Comment:
:clap: I like seeing this!

Reply:
Did you try LexiconCard.viewCardTitle? Not sure which viewCard* macro would match the themeName on the card.

Reply:
I didn't, I didn't see this macro, I will try later.

Reply:
LexiconCard.viewCardTitle works for this case. Will resend you a PR.